examples/quickstart/pom.xml

```diff
@@ -60,8 +60,17 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
-                <version>1.3.2</version>
                 <executions>
+                    <execution>
+                        <id>copy to swagger</id>
+                        <configuration>
+                            <executable>sh</executable>
+                            <arguments>
+                                <argument>-c</argument>
+                                <argument>cp ../../node_modules/@okta/openapi/dist/spec.yaml ../../src/swagger/api.yaml</argument>
```

Comment:
@VitaliiTytarenko-okta why is this plugin config needed in examples/quickstart pom.xml?

Reply:
In fact, it is not necessary, but I had to add this part here because `exec-maven-plugin` was already defined in this module. Therefore without these lines, I had an error during building due to a different package location.
But `<skip>` option can solve this problem.